manifests/ubuntu.pp

```diff
@@ -16,6 +16,13 @@
   Optional[String] $apt_keyserver = undef,
 ) inherits datadog_agent::params {
 
+  if $agent_version =~ /^[0-9]+\.[0-9]+\.[0-9]+$/ {
```

Comment:
This regex won't match versions like `7.15.0~beta.1`. A more complete regex that you can copy can be found in init.pp [1] although using such complex regexes makes me a bit sad :/

[1] https://github.com/DataDog/puppet-datadog-agent/blob/master/manifests/init.pp#L321

Reply:
I get why you are saying that. I'll make the change ^^.